### PR TITLE
Update Send all options 

### PR DESCRIPTION
**Pull request by neuodev**

### CODE REVIEW

**Comment by neuodev** [comment]
@yushih  There is a problem with the e2e test. even if I select the dropdown but it doesn't open

File: packages/yoroi-extension/features/support/webdriver.js [hunk 1]
```diff
@@ -247,6 +247,13 @@ function CustomWorld(cmdInput: WorldInput) {
       });
     }, index);
   };
+
+  this.clickElementByQuery = async (query) => {
+    await this.driver.executeScript(
+      `document.querySelector('${query}').click()`,
+    );
+  };
```
Comment (neuodev) [inline packages/yoroi-extension/features/support/webdriver.js:255]: Util we can use it whenever the normal `this.click` doesn't work

**Review by zuzunker — COMMENTED** [review]
/check

**Review by zuzunker — APPROVED** [review]
Works great!

**Pull request merged**